src/cegpy/trees/_staged.py

```diff
@@ -673,9 +760,11 @@ def calculate_AHC_transitions(
         :rtype: Dict
         """
 
-        self._store_params(prior, alpha, hyperstage)
+        self._store_params(prior, alpha, hyperstage, initial_staging)
 
-        loglikelihood, merged_situations = self._execute_ahc()
+        loglikelihood, merged_situations = self._execute_ahc(
+            initial_staging=initial_staging
+        )
```

Comment:
Why don't we pass `hyperstage` as an argument here?

Reply:
For some daft reason, you can pass it in as an argument, or it is pulled directly from `self`.
It is written to `self.hyperstage` in `self._store_params` so it should work as is. I think there needs to be some general packaging cleaning up. But that is out of scope for this PR.